README.md

```diff
@@ -1,211 +1,239 @@
+        - Keep **State** as **Enabled**
+        - Click on the **Add scope** button on the bottom to save this scope.
+
+#### Configure the service app (ms-identity-nodejs-webapi-azurefunctions) to use your app registration
+
+Open the project in your IDE (like Visual Studio or Visual Studio Code) to configure the code.
+
+> In the steps below, "ClientID" is the same as "Application ID" or "AppId".
+
+1. Open the `Function\auth.json` file.
+1. Find the key `clientID` and replace the existing value with the **application ID** (clientId) of the `ms-identity-nodejs-webapi-azurefunctions` application copied from the Azure portal.
+1. Find the key `tenantID` and replace the existing value with your Azure AD **tenant ID**.
+1. Find the key `audience` and replace the existing value with the **application ID** (clientId) of the `ms-identity-nodejs-webapi-azurefunctions` application copied from the Azure portal.
+1. Find the key `scope` and replace the existing value with the scope `access_as_user`.
+
+## Running the sample
+
+```console
+    cd ms-identity-nodejs-webapi-azurefunctions
+    func start
 ```
 
-## Contributing
+## Explore the sample
 
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
+1. The function app will run on `http://localhost:7071` when you test it locally.
+2. The function app will run on `https://<yournodejsfunction>.azurewebsites.net` when you run it deployed to Azure.
+
+You will need a **client** for calling the web API. Refer to the sample: [JavaScript single-page application calling a custom web API with MSAL.js 2.x using the auth code flow with PKCE](https://github.com/Azure-Samples/ms-identity-javascript-callapi).
+
+Once you install the **client** app, do:
+
+1. Open the `App\apiConfig.js` file.
+1. Find the key `Enter_the_Web_Api_Uri_Here` and replace the existing value with the coordinates of your web API (e.g. `http://localhost:7071/api`).
+1. Find the key `Enter_the_Web_Api_Scope_Here` and replace the existing value with the scopes for your web API (e.g. `api://e767d418-b80b-4568-9754-557f40697fc5/access_as_user`).
+
+> :information_source: Did the sample not work for you as expected? Then please reach out to us using the [GitHub Issues](../../../../issues) page.
+
+## About the code
+
+> - Describe where the code uses auth libraries, or calls the graph
+> - Describe specific aspects (e.g. caching, validation etc.)
+
+## Deployment
+
+### Deployment to Azure Functions
+
+There is one web project in this sample. To deploy it to **Azure Functions**, you'll need to:
+
+- create an **Azure Function**
+- publish the projects to the **Azure Functions**, and
+- update its client(s) to call the deployed function instead of the local environment.

```

Comment:
>environment [](start = 74, length = 11)

add these steps again

1. Open the `App\apiConfig.js` file.
1. Find the key `Enter_the_Web_Api_Uri_Here` and replace the existing value with the coordinates of your web API (e.g. `https://<yournodejsfunction>.azurewebsites.net`).

Reply:
this applies to the client SPA, which will be separate